embulk-core/src/main/java/org/embulk/exec/BulkLoader.java

```diff
@@ -512,7 +512,7 @@ public void doCleanup(ConfigSource config, ResumeState resume)
     private ExecutorPlugin newExecutorPlugin(BulkLoaderTask task)
     {
         return Exec.newPlugin(ExecutorPlugin.class,
-                task.getExecConfig().get(PluginType.class, "type", new PluginType("local")));
+                task.getExecConfig().get(PluginType.class, "type", PluginType.LOCAL));
```

Comment:
One reason is that I wanted to keep the creator method (`JsonCreator`) `private` until we really need that `public`.

In addition, that "local" type is substantially a constant. Making the `PluginType` value constant seemed more reasonable than making the string `"local"` constant.